src/test/resources/sql/voteIntTest.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO "film" ("name")
```

Comment:
use "`" instead of "  `film`

Reply:
I tried to find the answer for this, but it is not possible to use single quotes instead of double quotes...

Reply:
`